meetings/2018-09-14.md

```diff
@@ -0,0 +1,47 @@
+## Node.js Foundation User Feedback Meeting 2018-09-14
+## Links
+
+* **Recording**:https://www.youtube.com/watch?v=00YONf8MrIs
+* **GitHub Issue**: [91](https://github.com/nodejs/user-feedback/issues/91)
+* **Minutes Google Doc**: [Minutes](https://docs.google.com/document/d/1082uJlCb9ar9VThHI1ZRqfy9K2lpHtszSPGWkTfh53A/)
+
+## Present
+
+
+* User Feedback Members: @nodejs/user-feedback: Dan Shaw, Michael Dawson, Mihai Ene-Pietrosanu, Ahmad Nassri
```

Comment:
I think we could just make a bullet list.  I think Agiri is a member and Joe was looking to join as well.